### PR TITLE
UCT/IB: Logging bugfixes.

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4291/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@brminich pls take a look

File: src/uct/ib/mlx5/ib_mlx5_log.c [hunk 1]
```diff
@@ -274,23 +277,23 @@ static void uct_ib_mlx5_wqe_dump(uct_ib_iface_t *iface, enum ibv_qp_type qp_type
         int size = 1 << ((opmod & 7) + 2);
 
         if (opcode == MLX5_OPCODE_ATOMIC_MASKED_FA) {
-            add      = network_to_host(*(uint64_t*)(seg + 0), size);
-            boundary = network_to_host(*(uint64_t*)(seg + size), size);
+            add      = network_to_host(seg + 0 * size, size);
```
Comment (brminich) [inline src/uct/ib/mlx5/ib_mlx5_log.c]: what is the point in multiplying by zero?

Comment (yosefe) [inline src/uct/ib/mlx5/ib_mlx5_log.c]: mostly to be on the same code column with the next line

**Comment by yosefe** [comment]
@brminich @shamisp fixed

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4299/ for details (Mellanox internal link).

**Comment by shamisp** [comment]
👍

**Pull request merged**